node/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "log4js": "~0.6.9",
     "bower": "~1.3.12",
-    "lodash": "~2.2.1"
+    "lodash": "~2.2.1",
+    "bower-config": "^0.5.2"
```

Comment:
It's needed in BowerDomain. I had to install it manually. Thats why i've added it